client/src/Components/Footer/Footer.jsx

```diff
@@ -0,0 +1,33 @@
+import { Container } from "@mui/system";
+import React from "react";
+import { Grid } from "@mui/material";
+import { SocialIcon } from "react-social-icons";
+import "./Footer.css";
+export default function () {
+  return (
+    <Container maxWidth="lg" className="footer">
+      <Grid
+        container
+        justifyContent="space-between"
+        flexDirection={{ sm: "column", md: "row" }}
+      >
+        <div className="footerText">
+          <b>Contacts:</b>
+          <br></br>Address: Ivana Fedorova street, 10
```

Comment:
use self closing tag <br /> or it would be even better to wrap text lines in div tags